health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/ViewHierarchy.js

```
@@ -416,7 +416,7 @@ const ViewHierarchy = () => {
                                           <div>
                                             <div style={{ display: "flex", justifyContent: "space-between" }} key={index}>
                                               <div className="hierarchy-boundary-sub-heading2">
-                                                {`${t(( hierarchyType + "_" + hierItem?.boundaryType).toUpperCase().replace(/\s+/g, "_"))}`}
+                                                {`${t(( hierarchyType + "_" + hierItem?.boundaryType.trim().replace(/[\s_]+/g, '')).toUpperCase().replace(/\s+/g, "_"))}`}
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)